2. Web API now calls Microsoft Graph/TodoListService/TodoListService.csproj

```diff
@@ -10,7 +10,11 @@
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.AzureAD.UI" Version="2.2.0" />
     <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.Identity.Client" Version="2.7.1" />
+    <PackageReference Include="Microsoft.Identity.Client" Version="3.0.3-jmpreview" />
```

Comment:
A common practice is to have a 'dependencies.prop' file that defines the dependencies version.
For example the file would contain:
<MicrosoftIdentityClientVersion>3.0.3</MicrosoftIdentityClientVersion>

Then TodoListService.csproj would include dependencies.prop and write:
<PackageReference Include="Microsoft.Identity.Client" Version="$(MicrosoftIdentityClientVersion)"/>

This makes updating versions simpler and the same across projects.
For example see: 
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/dev5x/build/dependencies.props

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/dev5x/src/Microsoft.IdentityModel.Tokens/Microsoft.IdentityModel.Tokens.csproj

Reply:
Thanks Brent. We'll do that.